client/src/components/helpers/loader.tsx

```diff
@@ -5,23 +5,45 @@ import './loader.css';
 
 interface LoaderProps {
   fullScreen?: boolean;
-  timeout?: number;
+  loaderDelay?: number;
+  messageDelay?: number;
 }
-function Loader({ fullScreen, timeout }: LoaderProps): JSX.Element {
-  const [showSpinner, setShowSpinner] = useState(!timeout);
+function Loader({
+  fullScreen,
+  loaderDelay,
+  messageDelay
+}: LoaderProps): JSX.Element {
+  const [showSpinner, setShowSpinner] = useState(!loaderDelay);
+  const [showMessage, setShowMessage] = useState(!messageDelay);
   useEffect(() => {
-    let timerId: ReturnType<typeof setTimeout>;
-    if (!showSpinner) {
-      timerId = setTimeout(() => setShowSpinner(true), timeout);
+    if (loaderDelay) {
+      const timerId = setTimeout(() => setShowSpinner(true), loaderDelay);
+      return () => clearTimeout(timerId);
     }
-    return () => clearTimeout(timerId);
-  }, [setShowSpinner, showSpinner, timeout]);
+  }, [loaderDelay]);
+
+  useEffect(() => {
+    if (messageDelay) {
+      const timerId = setTimeout(() => setShowMessage(true), messageDelay);
+      return () => clearTimeout(timerId);
+    }
+  }, [messageDelay]);
+
   return (
     <div
       className={`fcc-loader ${fullScreen ? 'full-screen-wrapper' : ''}`}
       data-testid='fcc-loader'
     >
       {showSpinner && <Spinner name='line-scale-pulse-out' />}
+      {showMessage && fullScreen && (
+        <>
+          <br />
+          <p className='text-center'>
+            Looks like this is taking longer than usual, please try refreshing
+            the page.
+          </p>
+        </>
```

Comment:
If we add `gap`, the fragment and `br` element are useless.

```suggestion
        <p>
          Looks like this is taking longer than usual, please try refreshing
          the page.
        </p>
```

Reply:
I am pretty sure it comes from the bootstrap classes. These should be refactored collectively later.